services/eventservice/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM rust:latest
+
+WORKDIR /usr/src/app
```

Comment:
`/usr` is not a suitable folder for the repository. The folder for compiled user programs, not plain source code.

Reply:
What are the cons of holding source there? It is easier to hold sources and binaries together, as --output-dir for rust compiler is still unstable

Reply:
You may leave it anywhere. But we should respect established agreements in Linux.

Reply:
Well /usr/src folder is used exactly for source files, as its names states

Reply:
/usr/src:  Source files for different parts of the system, included with some packages for reference purposes.  Don't work here with your own projects, as files below /usr should be read-only except when installing software (optional).

Reply:
[Here](https://www.linuxquestions.org/questions/linux-newbie-8/usr-src-usage-133603/) and [here](https://qastack.ru/unix/10814/what-is-the-usr-local-src-folder-meant-for) people say that it is ok to put your files here
And I don't wanna put there my project, I wanna store there the only project that system exist 
Btw, I won't work with project there. I will copy it and leave read-only

Reply:
Ok, it's just a matter of agreement, let it be here...